packages/atlas/src/hooks/useNftTransactions.tsx

```diff
@@ -4,18 +4,27 @@ import { useCallback } from 'react'
 
 import { GetNftDocument, GetNftQuery, GetNftQueryVariables } from '@/api/queries'
 import { GetBidsDocument } from '@/api/queries/__generated__/bids.generated'
+import { Fee } from '@/components/Fee'
+import { NumberFormat } from '@/components/NumberFormat'
 import { NftSaleType } from '@/joystream-lib'
 import { useConfirmationModal } from '@/providers/confirmationModal'
-import { useJoystream } from '@/providers/joystream'
+import { useFee, useJoystream } from '@/providers/joystream'
 import { useTransaction } from '@/providers/transactions'
 import { useUser } from '@/providers/user'
+import { formatDateTime } from '@/utils/time'
 
 export const useNftTransactions = () => {
   const { memberId } = useUser()
   const { joystream, proxyCallback } = useJoystream()
   const handleTransaction = useTransaction()
   const [openModal, closeModal] = useConfirmationModal()
   const client = useApolloClient()
+  const { loading: withdrawFeeLoading, updateFullFeeHandler: updateWithdrawFee } = useFee(
+    'cancelNftBidTx',
+    undefined,
+    undefined,
+    true
+  )
```

Comment:
This will cause every user of `useNftTransaction` to calculate fee for this action. That means every NftTile and every VideoTile will calculate this fee (independently). That may not be a good idea. I think we should just create separate dialog component (like `ChangePriceDialog`) that can handle fee calculation and keep only the extrinsic calls in this file